### DISCUSSION TITLE
How to download Transformer model in Spacy?

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#9729] [Help: Installation] [answered]
Hey, I have been trying to install the Transfomer model using the spacy command but since I am using corporate server proxies are blocking it download deom .whl file. How can get .tar.gz file so that I can manually install it. Here is the image that shows followign error:
![image](https://user-images.githubusercontent.com/49562460/142911374-067d2033-405a-4feb-a370-d4ada030a8f7.png)

**Comment by polm** [comment, marked as answer]
You can download the pipeline from Github releases, [see here](https://github.com/explosion/spacy-models#manual-download-and-installation).
